includes/components/class-vendor.php

```diff
@@ -260,6 +173,27 @@ public function add_vendor_fields( $form_args, $form ) {
 
 				// Add fields.
 				foreach ( $vendor_form->get_fields() as $field_name => $field ) {
+
+					// Get user attribute name.
+					$user_attribute_name = hp\get_first_array_value(
```

Comment:
It may be easier to just check the field name in the form, if user attributes are added earlier (you can change the callback priority number where it's hooked if not), then if the field name already exists as an array key then skip adding the vendor field (we do this similarly in Attribute component, there are checks if the field name already exists in multiple places).